index.html

```diff
@@ -5,14 +5,78 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>Spotify Clone</title>
-    <!-- don't forget to link your styles -->
+    <link rel="stylesheet" href="./styles/spotify-clone.css"/>
   </head>
+
   <body>
-    Premium Discover Help Download Music for everyone. Spotify is now free on mobile, tablet and computer. Listen to the
-    right music, wherever you are. What’s on Spotify? Millions of Songs There are millions of songs on Spotify HD Music
-    Listen to music as if you were listening live Stream Everywhere Stream music on your smartphone, tablet or computer
-    It’s as yeezy as Kanye West. Search Know what you want to listen to? Just search and hit play. Browse Check out the
-    latest charts, brand new releases and great playlists for right now. Discover Enjoy new music every Monday with your
-    own personal playlist. Or sit back and enjoy Radio.
+    <header>
+      <img class="spotify-logo" src="./images/spotify-logo.png" alt="spotify logo" >
+      <nav>
+        <ul class="nav-links">
+          <li> <a href="#">Premium</a></li>
+          <li> <a href="#">Discover</a></li>
+          <li> <a href="#">Help</a></li>
+          <li> <a href="#">Download</a></li>
+        </ul>
+      </nav>
+
+    </header>
+     <section class= "box">
```

Comment:
Using `<main>` as a parent element for all these sections would be good practice